thrall/app/lib/ElasticSearch.scala

```diff
@@ -168,6 +182,10 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
     """.stripMargin
 
   // Create the exports key or add to it
+  private val addUsagesScript =
```

Comment:
Nitpicking, but could be named `replace` instead of `add` to differentiate from the exports one, which *does* add (append).